tests/integration/test_complete_run.cfg

```diff
@@ -104,22 +104,18 @@ years = "1850:1854:2", "1850:1854:4",
   climo_diurnal_subsection = "atm_monthly_diurnal_8xdaily_180x360_aave"
   climo_subsection = "atm_monthly_180x360_aave"
   diff_title = "Difference"
-  gauges_path = "/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/obs/time-series/GSIM/GSIM_catchment_characteristics_all_1km2.csv"
   ref_final_yr = 1851
   ref_name = "v2.LR.historical_0201"
   ref_start_yr = 1850
   ref_years = "1850-1851",
   reference_data_path = "/lcrc/group/e3sm/ac.forsyth2/zppy_test_complete_run_output/v2.LR.historical_0201/post/atm/180x360_aave/clim"
-  reference_data_path_climo_diurnal = "/lcrc/group/e3sm/ac.forsyth2/zppy_test_complete_run_output/v2.LR.historical_0201/post/atm/180x360_aave/clim_diurnal_8xdaily"
-  reference_data_path_tc = "/lcrc/group/e3sm/ac.forsyth2/zppy_test_complete_run_output/v2.LR.historical_0201/post/atm/tc-analysis_1850_1851"
-  reference_data_path_ts = "/lcrc/group/e3sm/ac.forsyth2/zppy_test_complete_run_output/v2.LR.historical_0201/post/atm/180x360_aave/ts/monthly"
-  reference_data_path_ts_rof = "/lcrc/group/e3sm/ac.forsyth2/zppy_test_complete_run_output/v2.LR.historical_0201/post/rof/native/ts/monthly"
   run_type = "model_vs_model"
   short_ref_name = "v2.LR.historical_0201"
   swap_test_ref = False
   tag = "model_vs_model"
   ts_num_years_ref = 2
   ts_subsection = "atm_monthly_180x360_aave"
+  years = "1852-1853",
```

Comment:
Now comparing 1852-1853 and 1850-1851, rather than comparing the same year sets. So, now there is actually a diff to plot.